appveyor.yml

```diff
@@ -0,0 +1,63 @@
+environment:
+  matrix:
+  - ARCH: "i686"
+    JULIA_URL: "https://julialangnightlies-s3.julialang.org/bin/winnt/x86/julia-latest-win32.exe"
+    
+  - ARCH: "x86_64"
+    JULIA_URL: "https://julialangnightlies-s3.julialang.org/bin/winnt/x64/julia-latest-win64.exe"
+
+skip_branch_with_pr: true
+
+skip_commits:
+# Add [av skip] to commit messages for docfixes, etc to reduce load on queue
+  message: /\[av skip\]/
+
+notifications:
+  - provider: Email
+    on_build_success: false
+    on_build_failure: false
+    on_build_status_changed: false
+
+# fetch repository as zip archive
+shallow_clone: true
+# set clone depth
+clone_depth: 5
+
+init:
+# Carriage returns are bad
+  - git config --global core.autocrlf input
+  - ps: "[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12"
+
+install:
+# Download and install most recent cygwin
+  - appveyor DownloadFile https://cygwin.com/setup-x86_64.exe -FileName C:\projects\cygwin-setup-x86_64.exe
+  - C:\projects\cygwin-setup-x86_64.exe -q -g -P make,mingw64-i686-gcc-core,mingw64-x86_64-gcc-core
+# Download most recent Julia Windows binary
+  - appveyor DownloadFile %JULIA_URL% -FileName C:\projects\julia-binary.exe
+# Set paths to only use this cygwin
+  - set PATH=C:\cygwin64\bin;"C:\Program Files\7-Zip"
+
+build_script:
+  - make CC=%ARCH%-w64-mingw32-gcc
```

Comment:
@tkelman it's way too complicated to figure out how to call make with the correct CC via cygwin sh, so unless you want to do it, I'm going to call it a day on this.